cmus.c

```diff
@@ -164,15 +154,17 @@ enum file_type cmus_detect_ft(const char *name, char **ret)
 	return FILE_TYPE_FILE;
 }
 
-void cmus_add(add_ti_cb add, const char *name, enum file_type ft, int jt, int force)
+void cmus_add(add_ti_cb add, const char *name, enum file_type ft, int jt,
+		int force)
```

Comment:
Some line length limit should be used. If you think 80 is too little, what about 90?